doc/lua_api.txt

```diff
@@ -3655,6 +3655,15 @@ These functions return the leftover itemstack.
           the Lua code generated will use that number of spaces as indentation
           instead of a tab character.
 
+* `minetest.read_schematic(schematic, options)`
+    * Returns a Lua table representing the schematic (see: Schematic specifier)
+    * `schematic` is the schematic to read (see: Schematic specifier)
+    * `options` is a table containing the following optional parameters:
+    * `write_yslice_prob`: integer value between 0 and 2, inclusive. 0 means
+      not to insert any write_yslice_prob table, 1 means to write values in this
```

Comment:
Would be more readable with each possible value on a separate line/list point.
How about using a string value here for easier understanding: `none`, `low` and `all`?

Reply:
Okay, fine for the string value, I'll change it